bingo/bingo-elastic/java/src/test/java/com/epam/indigo/elastic/FullUsageMoleculeTest.java

```diff
@@ -271,4 +270,13 @@ public void reactionTanimoto() {
             Assertions.fail("Exception happened during test " + exception.getMessage());
         }
     }
+
+    @Test
+    @DisplayName("Page size of 10000 should throw exception")
```

Comment:
fix display name to 1000

Reply:
display name is fine :) though fixed to 2000, since this is what used in test